fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -858,6 +907,7 @@ public void asyncToSync(@Nonnull CompletableFuture<Void> buildIndexFuture) {
         protected int maxRetries = DEFAULT_MAX_RETRIES;
         protected int recordsPerSecond = DEFAULT_RECORDS_PER_SECOND;
         private long progressLogIntervalMillis = DEFAULT_PROGRESS_LOG_INTERVAL;
+        private int increaseLimitAfter = -1;
```

Comment:
Is there a reason not to make this, say, 10 by default? I think we'd want to start increasing the limit in most cases if possible. Maybe I'm wrong; I suppose leaving it as off by default is the more conservative thing, and then it can be set to some non-negative value in the future?

We should probably document what its default behavior/value is, though.

Reply:
I document the default now. The reason that I went with not changing the default behavior is that would be a change in behavior, and I didn't think it is currently a bug, so figured maintaining backwards compatibility would be worth it, but I could pretty easily be persuaded otherwise.

Reply:
Okay, sounds good. I think in theory this could be tweaked later if it turns out this works well. My feeling is we want the long-term default to be that it increases given that if it decreases to a small number like 1 the index build can get effectively stuck forever for larger stores, but that sounds like something we could tweak later after this is more mature.